### PR TITLE
Support techs as a directory

### LINKED ISSUE #27: Support techs as a directory

### LINKED ISSUE #29: Output warning if file already exists

### PR DESCRIPTION
**Pull request by tadatuta**
Closes #27 
Closes #29

### CODE REVIEW

**Comment by tadatuta** [comment]
cc @vithar

File: .eslintrc.js [hunk 1]
```diff
@@ -9,7 +9,6 @@ module.exports = {
     extends: 'pedant',
     rules: {
         'no-unused-expressions': 'off',
-        'no-console': 'error',
```
Comment (vithar) [inline .eslintrc.js]: Irrelevant change

File: cli.js [hunk 2]
```diff
@@ -81,7 +81,7 @@ module.exports = function() {
                 block: opts.block[0],
                 elem: opts.elem && opts.elem[0],
                 modName: opts.mod && opts.mod[0],
-                modVal: opts.val ? opts.val[0] : Boolean(opts.mod),
```
Comment (vithar) [inline cli.js]: Irrelevant change

File: lib/create-entity.js [hunk 3]
```diff
@@ -4,19 +4,32 @@ var fs = require('fs'),
...
+                return resolve(fileName);
+            }
 
             mkdirp(path.dirname(fileName), function(err) {
                 if (err) return reject(err);
 
-                fs.writeFile(fileName, template(entity, bemNaming(options.naming)), function(error) {
-                    error ? reject(error) : resolve(fileName);
-                });
+                if (isFile) {
```
Comment (vithar) [inline lib/create-entity.js]: I suggest to inverse `if`:

```js
if (!isFile)
    return createTree(content, fileName, options).then(resolve).catch(reject);
```

**Pull request merged**